### ISSUE TITLE
NOT_NULL anko SQLite bad parsing

### ISSUE THREAD

**Issue by kolboch** [Kotlin/anko#485]
Hi, when using:
```
db?.createTable(FOO_TABLE,
                true,
                FOO_DATE to INTEGER + PRIMARY_KEY + NOT_NULL,
                FOO_TIME to INTEGER + DEFAULT("0"))
```
it is throwing exception where in message it appears it is converted like:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{... .MainActivity}: android.database.sqlite.SQLiteException: near "org": syntax error (code 1): , while compiling: 
CREATE TABLE IF NOT EXISTS `foo`
(foo_date INTEGER PRIMARY KEY org.jetbrains.anko.db.SqlTypeModifierImpl@420fa580, 
foo_time INTEGER DEFAULT 0);
```

**Comment by arslancharyev31** [comment]
Another duplicate of #469 and #373

**Comment by 4u7** [comment]
Fixed with a9fb477
